### ISSUE TITLE
websocket.userData [ 'Parse error: data.B is not iterable' ]

### ISSUE THREAD

**Issue by kasugasorata98** [jaggedsoft/node-binance-api#728]
The latest version of 0.13.0 makes the websocket.userData not work anymore

**Comment by gunar** [comment]
@kasugasorata98 thanks! this was very useful for me to open #732

**Comment by mr-smit** [comment]
any news ?

**Comment by luiztools** [comment]
No, it isn't @mr-smit. In 0.12.5 you could pass different callbacks for different updates from the stream and in 0.13.0 you can't anymore because of this wrong logic.

Any news @gunar ?

**Comment by gunar** [comment]
PR is ready just needs merging.
